src/editors/vscode/extensions.json

```diff
@@ -5,7 +5,7 @@
     "recommendations": [
         "ms-vscode.cpptools",
         "ms-python.python",
-        "chiehyu.vscode-astyle",
+        "xaver.clang-format",
```

Comment:
What is the advantage of using this separate extension for formatting? Did not yet try it out, but clang-format support seems to be also part of the C/C++ extension.

Reply:
I didn't get it to work properly. This extension kept using other formatters (apparently VS formatter or astyle) with some default formatting configuration. I didn't find a way to convince it to use clang-format and gave up eventually.

Reply:
In the hope to make this more robust, I switched off the `C/C++`'s formatting feature explicitly.